src/app/components/Link/Link.tsx

```diff
@@ -47,20 +47,15 @@ const Anchor = React.forwardRef<HTMLAnchorElement, AnchorProps>(
 			href={href || "#"}
 			{...props}
 		>
-			<span
-				className={cn("break-all border-b border-transparent transition-colors duration-200", {
-					"border-current border-dotted group-hover:border-transparent": isExternal,
-					"group-hover:border-current": !isExternal,
-				})}
-			>
+			<span className="break-all border-b border-transparent duration-200 group-hover:border-current">
```

Comment:
External links should have a dotted border.

Reply:
I talked to Oleg about this yesterday
https://app.slack.com/client/T28S90TCN/CLA4SFYE7/thread/CLA4SFYE7-1621411548.065400